dev_setup.sh

```diff
@@ -115,8 +121,8 @@ for var in "$@" ; do
 done
 
 if [[ $(id -u) -eq 0 && $opt_allowroot != true ]] ; then
-    echo 'This script should not be run as root or with sudo.'
-    echo 'If you really need to for this, rerun with --allow-root'
+    logger -f "$LOG_FILE" -s 'This script should not be run as root or with sudo.'
+    logger -f "$LOG_FILE" -s 'If you really need to for this, rerun with --allow-root'
```

Comment:
I'm not familiar with the shell logger but at least on a Ubuntu 20.04 LTS I seem to be getting a few errors trying to use this.

From a quick look:
- we need to ensure the log file exists first
- doesn't seem like you can set the log file and log to it in the same command?
  `logger: --file <file> and <message> are mutually exclusive, message is ignored`